components/server/src/authorization/relationship-updater.ts

```diff
@@ -79,6 +83,32 @@ export class RelationshipUpdater {
         // });
     }
 
+    private async createRelationsToWorkspaces(user: User): Promise<void> {
+        // Because the total amount of workspaces per user can be very large, we want to paginate through them.
+        const CHUNK_SIZE = 1000;
+        let minCreationTime = undefined;
+        while (true) {
+            const workspaces = await this.workspaceDB.findAllWorkspaces(0, CHUNK_SIZE, "creationTime", "ASC", {
+                ownerId: user.id,
+                minCreationTime,
+            });
+            if (workspaces.rows.length === 0) {
+                // We are done here
+                break;
+            }
+
+            await this.authorizer.bulkCreateWorkspaceInOrg(
```

Comment:
Can we make this call per org and user? I.e. `....(userId: string, orgId: string, workspaceIds: string[])` and call the method simply `addWorkspaces`?